source/_patterns/01-molecules/components/slide-with-text-inset.twig

```diff
@@ -0,0 +1,18 @@
+<div class="c-carousel__item--inset u-position--relative {{ carousel_item_class }}">
+
+  {% include '@atoms/images/picture.twig' with picture %}
+
+  <div class="c-carousel__item-text__wrap">
```

Comment:
Please remove the background color on this and add a class, `.u-theme--background-color-trans--dark` to this div.